.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,24 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+     - cron: "0 2 * * 2"
```

Comment:
We have the lowest number of upgrade PRs(6) on Tuesday hence I added the trigger schedule to be Tuesday 2:00am